<launch>
    <arg name="rand" default="10.0" />
    <arg name="num" default="5" />
    <arg name="detection" default="true" />
   <node pkg="swarm_localization" name="swarm_localization" type="swarm_localization_node" output="screen">
        <rosparam>
            force_freq: 1
            max_accept_cost: 10
            max_keyframe_num: 100
            min_keyframe_num: 5
            thread_num: 2
            min_kf_movement : 0.5
            init_xy_movement : 2.0
            init_z_movement : 1.0
        </rosparam>
        <param name="swarm_nodes_config" value="$(find swarm_localization)/config/swarm_nodes5.yaml" type="string" />
        <param name="initial_random_noise" value="$(arg rand)" type="double" />
    </node>
   <!-- <node pkg="swarm_localization" name="relative_fused_vicon_convert" type="relative_fused_vicon_convert.py" output="screen" /> -->

    <!-- <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find swarm_localization)/config/5drone-vicon.rviz" /> -->
    <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find swarm_localization)/config/5drone.rviz" />
    <node name="message_to_tf" pkg="message_to_tf" type="message_to_tf" output="log" args="/vins_estimator/imu_propagate" />
</launch>